modules/check_in/app/sidekiq/check_in/travel_claim_status_check_worker.rb

```diff
@@ -46,36 +51,84 @@ def claim_status(opts = {})
 
     # rubocop:disable Metrics/MethodLength
```

Comment:
@kanchanasuriya Can you move the case statement to another method and resolve the Metrics/MethodLength check.

Reply:
Please refer to the latest commit